builder/bflag.go

```diff
@@ -12,28 +13,32 @@ const (
 	stringType
 )
 
-type BuilderFlags struct {
+// BFlags contains all flags information for the builder
+type BFlags struct {
```

Comment:
why don't we call it just `Flags` so it will be `builder.Flags`? and also the functions below like `NewFlags` instead of `NewBFlags`